<launch>

	<!-- Machine -->
	<machine name="localhost" address="localhost" env-loader="/opt/ros/hydro/env.sh"/>
	<arg name="machine" default="localhost" />
	
	<!-- load terrain map configurations from YAML file to parameter server -->
	<rosparam file="$(find dwl_terrain)/config/terrain_map.yaml" command="load"/>
	
	<node pkg="dwl_terrain" type="terrain_map_server" name="terrain_map" output="screen" machine="$(arg machine)">
		<remap from="terrain_map" to="/terrain_map" />
		<remap from="octomap_binary" to="/octomap_full" />
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="world_frame" type="string" value="world" />
		<!-- Base frame of the robot -->
		<param name="base_frame" type="string" value="base_link" />
	</node>
	
</launch>
